packages/mcl/src/src/mcl/commands/ci_matrix.d

```diff
@@ -186,13 +187,20 @@ Package[] checkCacheStatus(Package[] packages)
     return packages;
 }
 
-export void print_table(string[] args)
+@(Command("print-table", "print_table").Description("Print a table of the cache status of each package"))
+struct print_table_args
+{
+}
+
+export int print_table(print_table_args args)
```

Comment:
Can we instead try supporting argument-less commands by providing an aptly argument-less function returning `void`, with `genSubCommandArgs` being updated to dynamically check whether or not each sub-command handler function has arguments/return value at all before taking them into account?
This also applies to all `_args` `struct`s annotated with `@(Command(" ").Description(" "))`.